CMakeLists.txt

```diff
@@ -339,6 +339,8 @@ find_package(gRPC ${VIAMCPPSDK_GRPC_VERSION_MINIMUM} CONFIG)
 if (gRPC_FOUND)
   get_target_property(VIAMCPPSDK_GRPC_CPP_PLUGIN gRPC::grpc_cpp_plugin LOCATION)
   set(VIAMCPPSDK_PROTOBUF_VERSION ${Protobuf_VERSION})
+  set(VIAMCPPSDK_PROTOBUF_VERSION_MAJOR ${Protobuf_VERSION_MAJOR})
```

Comment:
Buf's remote plugin versioning for protobuf only has major.minor, unlike grpc which has major.minor.patch. Compare the version drop-downs for:
- https://buf.build/protocolbuffers/cpp
- https://buf.build/grpc/cpp

So we need to break out major/minor for protobuf so we can interpolate them correctly. For the `find_package` case we get those variables set for us, but see below for the pkg-config case.